docs/specs/shapes.md

```diff
@@ -7,6 +7,37 @@ The graphical elements are divided in 4 categories:
 * [Styles](#shape-style), that define the visual appearance of shapes
 * [Modifiers](#modifier) alter the curves of the shapes
 
+## Shape Rendering Model
+
+The Lottie shape rendering model follows [AfterEffects' Shape Layer](
+https://helpx.adobe.com/after-effects/using/overview-shape-layers-paths-vector.html) semantics.
+
+### Grouping and Ordering Rules
+
+* **shapes** are rendered in reverse order (bottom->top)
+* **groups** offer a scoping mechanism for transforms, styles, and modifiers
+* **transforms** apply to all elements within scope, including group-nested elements
+* **styles** and **modifiers** apply to all preceding shapes within the current scope,
+  including group-nested shapes
+* when **multiple styles** apply to the same shape, the shape is rendered repeatedly for each style,
+  in reverse order
```

Comment:
Perhaps it would be clearer to say that the style is what causes the shapes to be rendered?